test/helpers/wrappers.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import "fmt"
+
+// PerfTest represents a type of test to run when running `netperf`.
+type PerfTest string
+
+const (
+	TCP_RR = PerfTest("TCP_RR")
```

Comment:
What does RR stand for? Would be nice to have descriptive name here.

Reply:
Done. Added links to netperf documentation as well.